roles/confluent.test/molecule/ccloud-kafka-rest/prepare.yml

```diff
@@ -1,22 +1,43 @@
 ---
-- name: Host Prerequisites
-  hosts: zookeeper:kafka_broker
+- name: Remove the Kafka Rest Groups from Inventory
+  hosts: kafka_rest
   gather_facts: false
   tasks:
-    - import_role:
-        name: confluent.common
-      tags: common
+    - name: Change kafka_rest Group Name in Inventory File
+      lineinfile:
+        path: "{{inventory_dir}}/ansible_inventory.yml"
+        regexp: "{{item}}:"
+        line: "{{item}}_ccloud:"
+      delegate_to: 127.0.0.1
+      run_once: true
+      loop:
+        - schema_registry
+        - kafka_connect
+        - kafka_rest
+        - ksql
+        - control_center
 
-- name: Zookeeper Provisioning
-  hosts: zookeeper
-  gather_facts: false
-  tasks:
-    - import_role:
-        name: confluent.zookeeper
+    - name: Refresh Inventory
+      meta: refresh_inventory
 
-- name: Kafka Broker Provisioning
-  hosts: kafka_broker
+- name: Converge
+  import_playbook: ../../../../all.yml
+
+- name: Kafka Rest Prepare
+  hosts: schema_registry_ccloud:kafka_connect_ccloud:kafka_rest_ccloud:ksql_ccloud:control_center_ccloud
   gather_facts: false
   tasks:
     - import_role:
-        name: confluent.kafka_broker
+        name: confluent.variables
+
+    - name: Copy CA Cert to Host
```

Comment:
Better test now uses another CA for component rest end points
Might need to manually test the ksqldb